runtime/cudaq/BaseIntegrator.h

```diff
@@ -0,0 +1,76 @@
+/****************************************************************-*- C++ -*-****
```

Comment:
with type name changed to base_integrator, would be best to have this file renamed to base_integrator.h